streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/IQv2IntegrationTest.java

```diff
@@ -86,7 +88,7 @@
 import static org.hamcrest.Matchers.matchesPattern;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@Timeout(600)
+@Timeout(1200)
```

Comment:
Do we actually need to bump the timeout here? Junit @Timeout annotation apply to a single parameter combination, so we should only bump this if the new protocol is in fact slower.

Same below